framework/src/modules/chain/init_steps/create_bus.js

```diff
@@ -0,0 +1,37 @@
+const changeCase = require('change-case');
+
+module.exports = async () => {
+	const Bus = function() {
```

Comment:
Prefer to consistently use `es6` class syntax.
Bus should be defined outside as class and used

Reply:
Yes it should, but in this iteration I just used the existing structure.

Reply:
I believe in future with the help of channels we will not be using this bus any more :) so for now leave it as it is.